src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -148,6 +148,19 @@ public void visitToken(DetailAST ast) {
             leftCurly = slistToken;
         }
 
+        // special check for TokenTypes.LITERAL_CASE
```

Comment:
> Also just to be clear what exactly do you mean by "UTs" ? unit tests, right !

Yes. As you added a new logic, you also should have implemented UTs to cover the changes. Our CIs control coverage rate and will not allow you to decrease its level. To generate coverage report you can use ```mvn clean cobertura:cobertura```. Coverage report will be located at ```target/site/cobertura/``` . To check whether the coverage rate is satisfied run ```mvn clean cobertura:check```.